agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -63,6 +76,7 @@ function help {
     echo "                [-cp | --collector_port <value>] [-s | --secure <value>] [-cc | --check_certificate <value>] \ "
```

Comment:
I would like that options order available options

Reply:
Talked with @alblasco about this and agreed in do so in a next PR